src/middlewared/middlewared/plugins/disk_/zfs_guid.py

```diff
@@ -49,7 +49,7 @@ async def sync_zfs_guid(self, pool_id_or_pool):
 
 async def devd_zfs_hook(middleware, data):
     if data.get("type") in (
-        "misc.fs.zfs.config_sync",
+        "sysevent.fs.zfs.config_sync",
```

Comment:
Did this event name was actually renamed in devd? Did you verify yourself?

Reply:
`sysevent.fs.zfs.config_sync` is the event that is now appearing in devd pipe if that's what you're talking about, I have not been examining actual ZFS code.